deps/patchelf.mk

```diff
@@ -9,7 +9,7 @@ $(SRCDIR)/srccache/patchelf-$(PATCHELF_VER)/source-extracted: $(SRCDIR)/srccache
 	touch -c $(SRCDIR)/srccache/patchelf-$(PATCHELF_VER)/configure # old target
 	echo 1 > $@
 
-$(BUILDDIR)/patchelf-$(PATCHELF_VER)/build-configured: $(SRCDIR)/srccache/patchelf-$(PATCHELF_VER)/source-extracted | $(LIBCXX_DEPENDENCY)
+$(BUILDDIR)/patchelf-$(PATCHELF_VER)/build-configured: $(SRCDIR)/srccache/patchelf-$(PATCHELF_VER)/source-extracted $(LIBCXX_DEPENDENCY)
```

Comment:
Ah, thanks for the correction. I put that change in a separate commit, so I'll just rebase out that one commit. Do you have a better idea for how to deal with this, @vtjnash?